pages/platform/kubernetes-k8s/getting-source-ip-behind-loadbalancer/guide.en-asia.md

```diff
@@ -167,18 +167,27 @@ ingress-nginx-controller   LoadBalancer   10.3.81.157   xxx.xxx.xxx.xxx   80:xxx
 
 Now you need to patch the Ingress controller to support the proxy protocol.
 
-#### 1. Get the list of the egress load balancer IPs
+> [!Warning]
+> Depends on your Kubernetes cluster is working with private network or not, the proxy protocol configuration differs.Follow the tutorial parts according to your setup.
+
+#### 1a. [PUBLIC NETWORK ONLY] Get the list of the egress load balancer IPs 
 
 ```bash
 kubectl get svc ingress-nginx-controller -n ingress-nginx -o jsonpath="{.metadata.annotations.lb\.k8s\.ovh\.net/egress-ips}"
 ```
 
 You should see something like this:
 
-<pre class="console"><code>$ kubectl get svc ingress-nginx-controller -n ingress-nginx -o jsonpath="{.metadata.annotations.lb\.k8s\.ovh\.net/egress-ips}"
+<pre class="console"><code>$ kubectl get svc ingress-nginx-controller -n ingress-nginx -o jsonpath="{.metadata.annotations.lb\.> k8s\.ovh\.net/egress-ips}"
 aaa.aaa.aaa.aaa/32,bbb.bbb.bbb.bbb/32,ccc.ccc.ccc.ccc/32,ddd.ddd.ddd.ddd/32
 </code></pre>
 
+#### 1b. [PRIVATE NETWORK ONLY] Get the list of the egress load balancer IPs
+
+We will use this [10.0.0.0/20] CIDR notation later.
+
+### Patching methods
+
 We can update the NGINX Ingress Controller configuration with manifest files or with Helm. Please choose one way or the other and follow the corresponding paragraph.
```

Comment:
"one way or the other " -> ca y est j'ai une musique en tete :-D

Reply:
Say thanks to Horacio :)